src/hooks/useDownload.ts

```diff
@@ -67,7 +66,11 @@ async function internal<T>({ src, headers, logger, dst }: DownloadOptions): Prom
 
     let txt = teal('downloading')
     if (sz) txt += ` ${gray(pretty_size(sz))}`
-    logger.replace(txt)
+    if (!json) {
+      logger.replace(txt)
+    } else {
+      console.log({status: "downloading"})
```

Comment:
I think `console.error` is going to be the correct move.

Reply:
agreed will update.